src/selectors/threadSelectors.js

```diff
@@ -1,13 +1,25 @@
 import { createSelector } from 'reselect';
 
+export const NonExistingRedditPostThreadId = 'NonExisting';
+
+
 // Gets the appropriate slice in the global app state
 const getThread = appState => appState.thread;
 
 // Gets Reply Form instance state slice
 export const getReplyForm = createSelector(
   getThread,
   (appState, threadId) => threadId,
-  (thread, threadId) => thread.replyForms[threadId]
+  (thread, threadId) => {
+    if (threadId === NonExistingRedditPostThreadId) {
```

Comment:
For non existing reddit post we always display root reply form (which is of course visible)